### DISCUSSION TITLE
Cannot connect to my paperless ngx

### DISCUSSION THREAD

**Discussion by UMA-111** [astubenbord/paperless-mobile#342] [General]
Hi Anton,
I set up a paperless ngx via docker on my synology. The web access does work without problems, but your mobile app refuse to connect via the reverse proxy URL.
It looks similar got the issue, which was reported in the review in google play store on 4th of december by Tobias Vorwerk.
When trying to connect it stops at "Authentifizieren". I only see turning circle the mentioned word. When I go to App Logs I see:
- Trying to log in  user@url -> In AuthenticationCubit.login()
- Adding new user user@url -> In AuthenticationCubit._adduser()
- Fetching bearer token from the server... -> In AuthenticationCubit._adduser()

A connection with the IP-address:port does work without any issue.

Any idea, why it doesn't work via the reverse proxy URL?

best regards,
Martin

**Comment by lagset** [comment]
Can not help directly on this, but as it may help troubleshooting: With my setup (paperless-ngx behind reverse proxy using a domain name, using https) it works.

However, I just tried to reproduce your issue and logged out of paperless-mobile and then tried to login again (with domain name as server address) and faced the issues you described (loading and stuck at "Authenticating").

After stopping paperless-mobile and emptying cache and storage of the app it worked again. So maybe try this, if you haven't?

**Comment by UMA-111** [comment]
Thanks for your reply, Valentin.
Yes, I cleared the cache all the time.
But I found the issue now. 
PAPERLESS_ALLOWED_HOSTS: needed to be added with the reverse-proxy URL. The it works like a charm.
That was no issue in the browser, but paperless mobile had problems with it.
Now it works as it should!
